keepassxc-browser/content/keepassxc-browser.js

```diff
@@ -1508,9 +1508,12 @@ kpxc.rememberCredentials = async function(usernameValue, passwordValue, urlValue
         if (credentialsList.length === 1) {
             usernameValue = credentialsList[0].login;
         } else if (credentialsList.length > 1) {
-            const index = await sendMessage('page_get_login_id');
-            if (index >= 0) {
-                usernameValue = credentialsList[index].login;
+            const uuid = await sendMessage('page_get_login_id');
```

Comment:
Sometimes we send `''` sometimes we leave the second parameter empty. Let's be consistent on that one.

Reply:
Are you mixing up `get` and `set`?

Reply:
Oof I totally am, that functionality is an innocent typo away!